everviz/plugins/_lineplot_plugin/lineplot_plugin.py

```diff
@@ -0,0 +1,37 @@
+import dash_html_components as html
+from uuid import uuid4
+
+from webviz_config import WebvizPluginABC
+
+from everviz.data.load_csv._get_data import get_data
+from everviz.plugins._lineplot_plugin._dash_components._lineplot_layout import (
+    _get_layout,
+)
+
+
+class LineplotPlugin(WebvizPluginABC):
+    def __init__(self, app, data_path, title="Lineplot"):
+        super().__init__()
+        self.title = title
+        self.graph_id = f"graph-{uuid4()}"
+        self.data_path = data_path
+        self.intital_y_data = "real_avg_obj"
```

Comment:
A class like `LineplotPlugin` is very general, while hardcoding the y-axis name really isn't.

I guess the `webviz-config` can take any arbitrary key names, I think we should move a lot of the specifications to the config file. That would make the plugins more reusable, and possible for other users to have their own as well.

Reply:
Good point, will address in lineplot PR.